docs/FAQ.md

```diff
@@ -307,7 +307,9 @@ Yes
 
 ### Are the R APIs for Spark supported?
 
-Yes, but we don't actively test them.
+Yes, but we don't actively test them. It is because the RAPIDS Accelerator hooks into Spark not at 
```

Comment:
We should have the same text for the Java APIs FAQ just below, as a user wanting to use the Java Spark APIs thought it would be risky to do so the way it was worded.

Reply:
Done.